content/docs/02-getting-started/04-svelte.mdx

```diff
@@ -44,7 +44,7 @@ Install `ai` and `@ai-sdk/openai`, the AI SDK's OpenAI provider.
   in the [providers](/providers) section.
 </Note>
 <div className="my-4">
-  <Tabs items={['pnpm', 'npm', 'yarn']}>
+  <Tabs items={['pnpm', 'npm', 'yarn', 'bun']}>
```

Comment:
bun tab missing

Reply:
Whoops, thanks

Reply:
still missing, you need to add a tab here